lms/djangoapps/django_comment_client/management/commands/export_discussion_participation.py

```diff
@@ -263,7 +262,9 @@ def _merge_user_data_and_social_stats(self, userdata, social_stats):
                 DiscussionExportFields.LAST_NAME: user.last_name,
             }
             stats = social_stats.get(user_id, self._make_social_stats())
-            result.append(utils.merge_dict(user_record, stats))
+            merged_dict = user_record.copy()
```

Comment:
We can optimize this as:
`user_record.update(stats)`
`result.append(user_record)`